src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -447,6 +447,18 @@ public string TargetedRuntimeVersion
             set { _targetedRuntimeVersionRawValue = value; }
         }
 
+        /// <summary>
+        /// If not null, serializes a cache to this location. This overrides the usual cache, so only use this if you will
+        /// not have access to the usual cache at the next build.
+        /// </summary>
+        public string CacheOutputPath { get; set; }
+
+        /// <summary>
+        /// If not null, uses this set of caches as inputs if RAR cannot find the usual cache in the obj folder. Typically
+        /// used for demos and first-run scenarios.
+        /// </summary>
+        public string[] CacheInputPaths { get; set; }
```

Comment:
```suggestion
        public string[] AssemblyInformationCaches { get; set; }
```

?

Reply:
I added Path to both names. I'm particularly sensitive to the possibility of confusing an object with the path to it right now, since one of the slnf bugs was that I said to use _solutionFilter instead of the path to the .slnf.